server/src/main/java/com/google/musicanalysis/site/MusixParamBuilder.java

```diff
@@ -0,0 +1,77 @@
+package com.google.musicanalysis.site;
+
+import java.net.URLEncoder;
+import java.nio.charset.StandardCharsets;
+
+/*
+    Builds parameter strings based off what the user wants.
+    This is structured the way it is becuase I will be adding more
+    to each function. I will clean it up later as well.
+*/
```

Comment:
nit: this should be formatted as javadoc, with asterisks before each line and two asterisks in the opening line

https://google.github.io/styleguide/javaguide.html#s7.1.1-javadoc-multi-line